msg/en_US.utf8/utils.msg

```diff
@@ -1309,3 +1310,27 @@ valid options:\n\
                                   for flashback; default: outputs only SQL statements for flashback\n\
     --oldest                      flashback is performed in chronological order for SQL statements executed from start time to end time;\n\
                                   default: latest\n
+
+$set 59 MSGCAT_UTIL_SET_MEMMON
+1 '-a(--all)' option cannot be used with other options.\n
+2 Please set system parameter 'memory_monitoring' in 'cubrid.conf'.\n\
+ It requires restarting cub_server.\n
+3 Invalid argument '%1$s' is set to '-m(--module)' option.\n
+4 -c(--tran-count) option must be used with -t(--transaction) option.\n
+59 memmon cannot run in standalone mode.\n
+60 \
+memmon: Provides memory usage information of a cub_server process.\n\
```

Comment:
To me, it would be nice to user logical name like 'the database server' iso process name like 'cub_server'. Is there any reason to use the cub_server?